lms/djangoapps/course_api/blocks/transformers/tests/test_student_view.py

```diff
@@ -0,0 +1,23 @@
+from openedx.core.lib.block_cache.block_structure import BlockStructureFactory
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+from ..student_view import StudentViewTransformer
+
+# TODO: Switch to SharedModuleStoreTestCase just as soon as you get that toy
+# course PR merged in.
```

Comment:
Sorry, that was a self-note. It's not blocking in the sense that it would hold up this merge, just a reminder that I could make the test faster if there were many more tests. I'm still doing my side hobby of making tests run faster, and I have a local branch with `test_module_render.py` changes that includes toy course generation for `SharedModuleStoreTestCase`. I think it's failing just one test now, but I've been too busy to go finish it off.